### ISSUE TITLE
Scrollbar appear on hover?

### ISSUE THREAD

**Issue by brian-hay** [daylilyfield/svrollbar#4]
Thanks for the component.

Is there a way to make the scrollbar appear on contents hover if 100% of contents aren't visible i.e. if there's stuff to scroll to?

At present it seems the scrollbar only appears when user attempts to scroll but there's no indication, by way of a visible scrollbar, that the area is scrollable.

**Comment by daylilyfield** [comment]
@brian-hay i am so sorry for my late response.  thank you for your feature request. 

> Is there a way to make the scrollbar appear on contents hover if 100% of contents aren't visible i.e. if there's stuff to scroll to?

there is no way to do so, but i have a plan to make scrollbar visible for first view/rendering (just like chrome does).

**Comment by madeleineostoja** [comment]
Just seconding that this would be a fantastic feature. My current solution is to always show scrollbars if there's content to scroll to, but it's pretty ugly compared to a more intelligently discoverable/visible scrollbar like you suggested

**Comment by oneezy** [comment]
I'd like this feature as well. 

Sometimes I move my mouse to the right side and "click" on the scrollbar track scroll up/down the page. With the way svrollbar is currently implemented there's no way to do that because it completely mounts and destroys (adds/removes) scrollbar to the DOM when you hover. It would be more ideal if were _always there_ but it was just set to `opacity: 0` if there was no scrolling. But upon scroll or track hover, it would fade in and act more like default scroll behavior

**Comment by daylilyfield** [comment]
i have just released v0.11.0 that has a new feature `initiallyVisible`.  enabling this property, scrollbar is visible if content is scrollable and content does not scrolled yet.

**Comment by madeleineostoja** [comment]
Amazing! Thank you for the quick fix!

**Comment by madeleineostoja** [comment]
Oh true, I hadn't tested `initiallyVisible` yet. Couldn't you achieve that behaviour with `alwaysVisible` and your own CSS on the scrollbar? Though I do think that `alwaysVisible` should only apply if the area is scrollable (ie: content > 100% height of container), which isn't the current behaviour

**Comment by oneezy** [comment]
> _Couldn't you achieve that behaviour with alwaysVisible and your own CSS on the scrollbar?_

yeah i suppose that would be a pretty simple fix @madeleineostoja though there would be that 100% height issue you mentioned. 

. . .

I also said this above but it doesn't really apply to this particular issue,,
> _Sometimes I move my mouse to the right side of the viewport and "click" anywhere on the scrollbar track to scroll up/down the page._

I guess it would be the next step though 😜 
I can create a separate issue/ feature request for that if you'd like

**Comment by madeleineostoja** [comment]
Made a more specific issue for that https://github.com/daylilyfield/svrollbar/issues/9
